Testscripts/Windows/DiskHotAddRemove-Serial.ps1

```diff
@@ -45,7 +45,7 @@ function Main {
                 Write-LogInfo "$count - Successfully attached an empty data disk of size $diskSizeinGB GB to VM"
             } else {
                 $testResult = $resultFail
-                throw "Fail to attach�an empty data disk of size $diskSizeinGB GB to VM"
+                throw "Fail to attach�an empty data disk of size $diskSizeinGB GB to VM"
```

Comment:
Expected symbol?

Reply:
fixed in 05932f86e8b954c3b1a541ffc483c96ca1036710